<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_Favoris</class>
 <widget class="QMainWindow" name="view_Favoris">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>831</width>
      <height>471</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>241</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Mes Favoris&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layout_add" native="true">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>58</y>
      <width>232</width>
      <height>290</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="edit_value">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>+ / - €</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="list_pmt">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_desc">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="edit_ref">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Référence</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="list_categories">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="list_magasins">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Ajouter aux favoris</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1097</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAccueil">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionExporter_la_base_de_donnees"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>Navigation</string>
    </property>
    <addaction name="actionAccueil"/>
    <addaction name="separator"/>
    <addaction name="actionComptes"/>
    <addaction name="actionMagasins"/>
    <addaction name="actionCategories"/>
   </widget>
   <addaction name="menuAccueil"/>
   <addaction name="menuNavigation"/>
  </widget>
  <action name="actionAccueil">
   <property name="text">
    <string>Accueil</string>
   </property>
  </action>
  <action name="actionCategories">
   <property name="text">
    <string>Catégories</string>
   </property>
  </action>
  <action name="actionMagasins">
   <property name="text">
    <string>Magasins</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionRecherche">
   <property name="text">
    <string>Recherche</string>
   </property>
  </action>
  <action name="actionExporter_la_base_de_donnees">
   <property name="text">
    <string>Exporter la base de données</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Imprimer</string>
   </property>
  </action>
  <action name="actionFavoris">
   <property name="text">
    <string>Favoris</string>
   </property>
  </action>
  <action name="actionComptes">
   <property name="text">
    <string>Comptes</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
